File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.py
def solution(arr1, arr2):
    answer = []
    result = []
    for i in range(0, len(arr1)):
        for j in range(0, len(arr2[0])):
            result.append(arr1[i][j] + arr2[i][j])
        answer.append(result)
        result = []
    return answer

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.js
function solution(arr1, arr2) {
    var answer = [];
    for(let i = 0; i < arr1.length; i++){
        let arr = arr1[i];
        let arrs = [];
        
        for(let j = 0; j < arr.length; j++){
            arrs.push(arr1[i][j] + arr2[i][j]);
            
        }
        answer.push(arrs);
    }
    return answer;
}